doc/doxygen/src/Glossary.dox

```diff
@@ -180,13 +180,16 @@
  * user-facing Digital Content Creation application or a pipeline
  * tool or script.
  *
- * The purpose of OpenAssetIO is to standardize the interaction between a host
- * and an @ref asset_management_system in order to persis or query data.
- * A host communicates with a @ref manager via the @ref openassetio.hostApi.Manager.Manager class.
+ * The purpose of OpenAssetIO is to standardize the interaction between
+ * a host and an @ref asset_management_system in order to persist or 
+ * query data.
+ * A host communicates with a @ref manager via the
+ * @fqref{hostApi.Manager} "Manager" class.
  *
  * > Note: The Manager class is an abstraction, that takes care of
  * > sundry housekeeping and helps manage session state. Internally it
- * > holds an instance of the @ref openassetio.managerApi.ManagerInterface.ManagerInterface
+ * > holds an instance of the
+ * > @fqref{managerApi.ManagerInterface}.ManagerInterface
```

Comment:
[fix] This links to the `ManagerInterface` constructor, probably not what was intended.

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1058/commits/948efe5518f4da207ec3438f17531c456b524965